conf/log4j.properties

```diff
@@ -34,6 +37,14 @@ log4j.appender.MASTER_LOGGER.MaxBackupIndex=100
 log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
 
+# Remote appender for Master
```

Comment:
I suggest grouping all of the logserver configuration in one place at the bottom since they will generally be configured together

Reply:
Done.